toolbox/fdc3-compliance/src/test/fdc3.addContextListener.ts

```diff
@@ -0,0 +1,21 @@
+import { Listener } from "@finos/fdc3";
+
+describe("fdc3.addContextListener", () => {
+  let listener: Listener;
+
+  afterEach(async () => {
+    listener.unsubscribe();
```

Comment:
We could get rid of the async, add a check for whether the listener's type is !== undefined and finally unset the listener after unsubscribing.